src/procs/arguments.md

```diff
@@ -0,0 +1,34 @@
+# Arguments
+
+## ⚠⚠ WIP PAGE ⚠⚠
+
+The parameters that are found within the paraentheses of a proc are know as `arguments`. This allows you to make more diverse procs and helps to prevent rewriting similar code as different values can be used in the same proc with arguments.
```

Comment:
```suggestion
The parameters that are found within the paraentheses of a proc are known as `arguments`. This allows you to make more diverse procs and helps to prevent rewriting similar code as different values can be used in the same proc with arguments.
```